pbuilder-hookdir/B90lintian

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Hook to run lintian inside the cowbuilder environment.
+#
+
+BUILDDIR="${BUILDDIR:-/tmp/buildd}"
+
+if [ "${LINTIAN:-}" != "true" ] ; then
+  echo "Skipping lintian (LINTIAN is not 'true')"
+  exit 0
+fi
+
+if [ -n "${LINTIAN_OPTIONS:-}" ] ; then
+  echo "*** Using provided LINTIAN_OPTIONS $LINTIAN_OPTIONS ***"
+fi
+
+set -ex -o pipefail
+apt-get -y "${APTGETOPT[@]}" install lintian
+lintian --version
+
+PACKAGE_NAME=$(basename "$(realpath "$BUILDDIR"/*/debian/..)")
```

Comment:
Sometime `realpath` will not be available. It is provided by `coreutils`:

 $ dpkg -S /usr/bin/realpath
 coreutils: /usr/bin/realpath

There might be a way to extract the package name without `realpath` though. Else the hook has to install the `coreutils` package as well.